lib/schnorr.py

```diff
@@ -161,7 +161,7 @@ def sign(privkey, message_hash):
         pubbytes = point_to_ser(pubpoint, comp=True)
 
         k = nonce_function_rfc6979(order, privkey, message_hash,
-                                   algo16=b'Schnorr+SHA256\x20\x20')
+                                   algo16=b'Schnorr+SHA256\x20\x20',ndata=ndata)
```

Comment:
Is the addition of the default `ndata=b''` equivalent to the previous call?

Reply:
Yes, see https://github.com/Electron-Cash/Electron-Cash/blob/master/lib/schnorr.py#L99  

If there is nothing present, it just becomes empty (byte) string

Reply:
Right, in that execution path in the python that's ok -- but on the secp side it is not the same thing.